sched/generators.go

```diff
@@ -82,7 +82,7 @@ func GenRandomTask(rng *rand.Rand) TaskDefinition {
 		envVarsMap[fmt.Sprintf("env%d", j)] = testhelpers.GenRandomAlphaNumericString(rng)
 	}
 
-	timeout := time.Duration(rng.Int63n(1000))
+	timeout := time.Duration(rng.Int63n(1000)) * time.Second
```

Comment:
Before, the timeout would always be <1000ns, which is 1 us, which is too short for anything to succeed.